core/src/test/scala/unit/kafka/server/GroupCoordinatorBaseRequestTest.scala

```diff
@@ -38,6 +46,12 @@ class GroupCoordinatorBaseRequestTest(cluster: ClusterInstance) {
 
   private def controllerServers(): Seq[ControllerServer] = cluster.controllers().values().asScala.toSeq
 
+  protected def securityProtocol: SecurityProtocol = SecurityProtocol.PLAINTEXT
```

Comment:
we are always passing securityProtocol, listenerName. looks the are not necessary here